docs/BUILD.md

```diff
@@ -39,6 +39,21 @@ Build the QT frontend for MacOS:
 Build artifacts:
 * `frontends/qt/build/osx/BitBox.app`
 
+### Signing & Notarization
+
+Requires Xcode 10+ and macOS 10.13.6+.
+
+```
+$ cd frontends/qt/build/osx/
+$ # Sign with hardened runtime:
+$ codesign -f --deep --timestamp -o runtime --entitlements ../../resources/MacOS/entitlements.plist -s CODESIGN_IDENTITY BitBox.app
```

Comment:
Although that `codesign` command works, I would consider adding the `--strict` flag, which is recommended in the Apple notarization documentation.

It's worthwhile to note that using the `--timestamp` flag requires a good internet connectivity, due to external timestamp validation. If the connectivity is shaky, the code signing without `--strict` will still succeed without warning, but the result cannot be notarized. With the option `--strict`, the code signing will fail, which is preferrable in this case.